examples/skip_plot_functionality.py

```diff
@@ -0,0 +1,51 @@
+"""
+=====================
+Adding extra features
+=====================
+
+sunkit-pyvista also extends features from `sunpy`, producing
+them in 3D.
```

Comment:
```suggestion
sunkit-pyvista also extends features from `sunpy`, producing them in 3D.
```
What features?

Reply:
The examples listed out here? Should they be called something else?

Reply:
It needs to explain what these features are.